Coupled-Cluster/spin_free_CC/helper_cc.py

```diff
@@ -1579,6 +1580,192 @@ def linresp(self):
 
 # End cclinresp class
 
+
+class helper_cceom(object):
```

Comment:
Classes should be PascalCase not snake case.

Reply:
Got it, I was just matching the other helper classes above. Do you want me to change that on all the `helper_*` classes while I'm at it?

Reply:
That would be great, yes.